### PR TITLE
Docs: Corrected incorrect instances of (setup|set up)

### PR DESCRIPTION
**Pull request by philnichol**
**What this PR does / why we need it**:
Corrects incorrect usage of (setup|set up) in the docs
**Which issue(s) this PR fixes**:
No linked issue

**Special notes for your reviewer**:
I ran this in Grafana and they suggested I do it here too.
**Checklist**
- [x] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: docs/sources/architecture/_index.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ value is larger than the hash of the stream. When the replication factor is
 larger than 1, the next subsequent tokens (clockwise in the ring) that belong to
 different ingesters will also be included in the result.
 
-The effect of this hash set up is that each token that an ingester owns is
+The effect of this hash setup is that each token that an ingester owns is
```
Comment (oddlittlebird) [inline docs/sources/architecture/_index.md:91]: This version of "setup" is a noun. Please revert this change.

File: docs/sources/clients/aws/eks/_index.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ title: EKS
 ---
 # Sending logs from EKS with Promtail
 
-In this tutorial we'll see how setup promtail on [EKS][eks]. Amazon Elastic Kubernetes Service (Amazon [EKS][eks]) is a fully managed Kubernetes service, using Promtail we'll get full visibility into our cluster logs. We'll start by forwarding pods logs then nodes services and finally Kubernetes events.
+In this tutorial we'll see how set up promtail on [EKS][eks]. Amazon Elastic Kubernetes Service (Amazon [EKS][eks]) is a fully managed Kubernetes service, using Promtail we'll get full visibility into our cluster logs. We'll start by forwarding pods logs then nodes services and finally Kubernetes events.
```
Comment (oddlittlebird) [inline docs/sources/clients/aws/eks/_index.md:6]:
```suggestion
In this tutorial we'll see how to set up Promtail on [EKS][eks]. Amazon Elastic Kubernetes Service (Amazon [EKS][eks]) is a fully managed Kubernetes service, using Promtail we'll get full visibility into our cluster logs. We'll start by forwarding pods logs then nodes services and finally Kubernetes events.
```

**Review by oddlittlebird — CHANGES_REQUESTED** [review]
Thanks! Couple of editing suggestions.

File: docs/sources/architecture/_index.md (hunk 1 above)
Comment (philnichol) [reply docs/sources/architecture/_index.md:91]: Obviously happy to revert this, but shouldn't it be a noun?
Or should it be "The effect of this hash being set up is that..."?

Comment (philnichol) [reply docs/sources/architecture/_index.md]: gotcha, apologies! Cheers for the feedback, I've reverted that change

Comment (oddlittlebird) [reply docs/sources/architecture/_index.md]: If you are ready for another review, I suggest clicking "Re-request review." People don't always notice comments, but the request for review sends a separate (and more visible) notification.

Comment (philnichol) [reply docs/sources/architecture/_index.md]: Thanks, didn't know that!

**Review by oddlittlebird — APPROVED** [review]
LGTM! Thanks for contributing to our docs!

**Comment by owen-d** [comment]
Thanks!
